<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QDialog" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1151</width>
    <height>481</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string notr="true">시험1</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QTabWidget" name="tab_kind">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>1111</width>
     <height>461</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="first_tab">
    <attribute name="title">
     <string>(1) 검토용파일 제작</string>
    </attribute>
    <widget class="QLabel" name="label_third">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>611</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>3. 검토용파일 이름을 입력해주세요.</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="QTextEdit_test_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>511</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;ex)&lt;/span&gt; 함수의 극한~함수의 그래프 (210828)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_grade">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>학년</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <widget class="QRadioButton" name="radioButton_grade1">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>고1</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_grade2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>고2</string>
      </property>
     </widget>
    </widget>
    <widget class="QTextEdit" name="QTextEdit_excel_directory">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>501</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;pre style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'맑은 고딕'; font-size:9.8pt; color:#080808; background-color:#ffffff;&quot;&gt;엑셀파일의 경로를 입력해주면 됩니다&lt;/span&gt;&lt;span style=&quot; font-family:'JetBrains Mono','monospace'; font-size:9.8pt; color:#080808;&quot;&gt;.&lt;br/&gt;Ex) excel = r&amp;quot;C:\Users\Season\Desktop\&lt;/span&gt;&lt;span style=&quot; font-family:'맑은 고딕'; font-size:9.8pt; color:#080808;&quot;&gt;내신주문서&lt;/span&gt;&lt;span style=&quot; font-family:'JetBrains Mono','monospace'; font-size:9.8pt; color:#080808;&quot;&gt;.xlsx&amp;quot;&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-family:'맑은 고딕'; font-size:9.8pt; color:#080808;&quot;&gt;경로를 쉽게 따오는 방법은 해당 파일을 누르고 &lt;/span&gt;&lt;span style=&quot; font-family:'JetBrains Mono','monospace'; font-size:9.8pt; color:#080808;&quot;&gt;(Shift+F10)&lt;/span&gt;&lt;span style=&quot; font-family:'맑은 고딕'; font-size:9.8pt; color:#080808;&quot;&gt;을 누르고 &lt;/span&gt;&lt;span style=&quot; font-family:'JetBrains Mono','monospace'; font-size:9.8pt; color:#080808;&quot;&gt;A&lt;/span&gt;&lt;span style=&quot; font-family:'맑은 고딕'; font-size:9.8pt; color:#080808;&quot;&gt;를 누르면 경로가 자동으로 복사됩니다&lt;/span&gt;&lt;span style=&quot; font-family:'JetBrains Mono','monospace'; font-size:9.8pt; color:#080808;&quot;&gt;.&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-family:'맑은 고딕'; font-size:9.8pt; color:#080808;&quot;&gt;만약 엑셀파일을 내신주문서 대신 해당 부분만 따로 떼서 하실 경우 &lt;/span&gt;&lt;span style=&quot; font-family:'JetBrains Mono','monospace'; font-size:9.8pt; color:#080808;&quot;&gt;&amp;quot;&lt;/span&gt;&lt;span style=&quot; font-family:'맑은 고딕'; font-size:9.8pt; color:#080808;&quot;&gt;고&lt;/span&gt;&lt;span style=&quot; font-family:'JetBrains Mono','monospace'; font-size:9.8pt; color:#080808;&quot;&gt;1 2021&amp;quot; &lt;/span&gt;&lt;span style=&quot; font-family:'맑은 고딕'; font-size:9.8pt; color:#080808;&quot;&gt;또는 &lt;/span&gt;&lt;span style=&quot; font-family:'JetBrains Mono','monospace'; font-size:9.8pt; color:#080808;&quot;&gt;&amp;quot;&lt;/span&gt;&lt;span style=&quot; font-family:'맑은 고딕'; font-size:9.8pt; color:#080808;&quot;&gt;고&lt;/span&gt;&lt;span style=&quot; font-family:'JetBrains Mono','monospace'; font-size:9.8pt; color:#080808;&quot;&gt;2 2021&amp;quot; &lt;/span&gt;&lt;span style=&quot; font-family:'맑은 고딕'; font-size:9.8pt; color:#080808;&quot;&gt;로 시트 이름을 바꾸어 주세요&lt;/span&gt;&lt;span style=&quot; font-family:'JetBrains Mono','monospace'; font-size:9.8pt; color:#080808;&quot;&gt;.&lt;/span&gt;&lt;/pre&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="toolTipDuration">
      <number>5000</number>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;C:\Users\Season\Desktop\자동화\임시 엑셀.xls&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_fourth">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>691</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>4. 제작!</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_second">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>551</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>2. 학년을 선택해주세요.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_first">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>611</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>1. 주문서(엑셀) 파일 경로를 입력해주세요.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_execute">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>380</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>검토용파일 제작하기!</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressbar">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>400</y>
       <width>451</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_progress">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>0</y>
       <width>451</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_find_excel">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>70</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>찾아보기(...)</string>
     </property>
    </widget>
    <zorder>groupBox_grade</zorder>
    <zorder>textBrowser_progress</zorder>
    <zorder>QTextEdit_test_name</zorder>
    <zorder>label_fourth</zorder>
    <zorder>progressbar</zorder>
    <zorder>QTextEdit_excel_directory</zorder>
    <zorder>label_second</zorder>
    <zorder>label_first</zorder>
    <zorder>label_third</zorder>
    <zorder>pushButton_execute</zorder>
    <zorder>pushButton_find_excel</zorder>
   </widget>
   <widget class="QWidget" name="second_tab">
    <attribute name="title">
     <string>(2) 검토용파일 반영/출처 표시</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_grade_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>학년</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <widget class="QRadioButton" name="radioButton_grade1_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>고1</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_grade2_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>고2</string>
      </property>
     </widget>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_progress_2">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>0</y>
       <width>451</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="QTextEdit_test_name_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>511</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;ex)&lt;/span&gt; 함수의 극한~함수의 그래프 (210828)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_fourth_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>401</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>4. 반영!</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressbar_2">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>400</y>
       <width>451</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTextEdit" name="QTextEdit_excel_directory_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>501</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Gulim'; font-weight:600;&quot;&gt;ex)&lt;/span&gt;&lt;span style=&quot; font-family:'Gulim';&quot;&gt; C:\Season\검토용파일.xls&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_second_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>551</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>2. 학년을 선택해주세요.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_first_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>691</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>1. 검토용파일 경로를 입력해주세요.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_execute_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>380</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>반영 진행</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_third_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>691</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>3. 검토용파일 이름을 입력해주세요.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_find_excel_2">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>70</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>찾아보기(...)</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
